flow/project.py

```diff
@@ -5032,9 +5050,9 @@ class MyProject(FlowProject):
             help="Do not actually execute commands, just show them.",
         )
         execution_group.add_argument(
-            "--progress",
-            action="store_true",
-            help="Display a progress bar during execution.",
+            "--hide_progress",
+            action="store_false",
```

Comment:
Trying to make sure I understand here -- why is this `store_false` and the other line above is `store_true`? Does the progress behavior match the behavior prior to this PR?

Reply:
This did change behavior and has been removed.